ci-operator/step-registry/cucushift/installer/check/aws/custom-ami/cucushift-installer-check-aws-custom-ami-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: cucushift-installer-check-aws-custom-ami
+  from: upi-installer
```

Comment:
If we do not rely on openshift-installer any more, we can use a specific tag name of image to run the step.